caller-workflow-templates/cintegration-ecs-explorer-caller.yml

```diff
@@ -35,4 +35,6 @@ jobs:
       next-public-graasp-perform-host:
       next-public-graasper-id:
       next-public-published-tag-id:
+      next-public-s3-files-host:
+      next-public-sentry-dsn:
```

Comment:
Could we add sentry in the app's workflow? (I didn't check if it was already the case or not)